packages/playwright-core/src/server/chromium/crNetworkManager.ts

```diff
@@ -158,10 +175,23 @@ export class CRNetworkManager {
     });
   }
 
+  // Service Worker Network.requestWillBeSent event fires on the parent frame session,
+  // but the Fetch.requestPaused comes in on the Service Worker session.
+  // We assume these came in the the same session (on the Service Worker), so shuffle things around.
+  // In the future, this may be resolved; see https://crbug.com/1304536
+  _maybeAdoptServiceWorkerMainRequest(id: string) {
+    const event = this._parentManager?._requestIdToRequestWillBeSentEvent.get(id);
```

Comment:
Let's check the `'service-worker'` header and ignore this request inside the page's network manager.